builtin/rebase.c

```diff
@@ -835,6 +836,9 @@ int cmd_rebase(int argc, const char **argv, const char *prefix)
 	if (ignore_whitespace)
 		strbuf_addstr(&options.git_am_opt, " --ignore-whitespace");
 
+	if (isatty(2) && options.flags & REBASE_NO_QUIET)
+		cmd_format_patch(1, &progress, "git_format_patch_opt");
+
```

Comment:
No, this is not correct. You will have to use the exact same strategy as for `git_am_opt`. You will have to add a field to `rebase_options`, and you will have to handle it via `add_var()`.